google-cloud-storage/src/test/resources/com/google/cloud/storage/conformance/retry/hello-world.txt

```diff
@@ -0,0 +1 @@
+Hello, World!!!
```

Comment:
Do we need this checked in? I think python uses a `NamedTemporaryFile`, not sure if java has an equivalent.

Reply:
Some of the java api accepts a file path so I did this file to satisfy that requirement. If we don't want the test resource it could be refactored.

Reply:
Ah yeah, I meant that python has the same requirement but they use [NamedTemporaryFile](https://docs.python.org/3/library/tempfile.html#tempfile.NamedTemporaryFile) to create a valid file path which gets generated and cleaned up each time. Not sure if there is something that Java offers.

Reply:
It's something we'd have to supply.